cluster-state-service/handler/store/datastore.go

```diff
@@ -158,7 +181,17 @@ func (datastore etcdDataStore) stream(ctx context.Context, keyPrefix string, kvC
 			}
 			resetStreamIdleTimer(streamIdleTimer)
 			for _, ev := range event.Events {
-				kv := map[string]string{string(ev.Kv.Key): string(ev.Kv.Value)}
+				// Skip empty events, such as Etcd deletes.
```

Comment:
I'm not sure.. Maybe introduce a status? Lets add a TODO for this and come back to it. We may want to ignore it like this code is already doing. But lets revisit later.